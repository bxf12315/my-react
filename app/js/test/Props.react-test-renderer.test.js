import App from "../components/Props.js";
import React from 'react';
import renderer from 'react-test-renderer';
import {render, fireEvent, cleanup} from 'react-testing-library';
afterEach(cleanup);

test('App test',()=>{
    const app = renderer.create(
        <App name={"xiabai"} url={"http://www.redhat.com"} nickname={"xiaofeng"} />
     );
     let tree = app.toJSON();
     expect(tree).toMatchSnapshot();
   
});

