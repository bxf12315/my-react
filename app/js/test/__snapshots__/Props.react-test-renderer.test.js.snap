// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test 1`] = `
<div>
  <h1
    style={
      Object {
        "color": "red",
        "textAlign": "center",
      }
    }
  >
    websit: 
    xiabai
  </h1>
  <h1>
    websit: 
    http://www.redhat.com
  </h1>
  <h1>
     name: 
    xiaofeng
  </h1>
</div>
`;
